src/ResourceManager/Subscription/AzureRM.Subscription.Preview.Netcore.psd1

```diff
@@ -0,0 +1,133 @@
+#
```

Comment:
@MiYanni I'm not sure if we need to convert this module to .NET Core since it's a preview module

Reply:
I've converted every module, preview or otherwise, as these projects are intended to be the 'only' projects we'll use at some point in the not-so-distant future. So, in this case, just being thorough.